docs/installing-northstar/using-northstar/playing-on-linux.md

```diff
@@ -155,15 +77,15 @@ Running the game on fullscreen through Linux might lead to a black screen preven
 
 For more info and proposed fixes, refer to [this issue thread on Github](https://github.com/R2Northstar/Northstar/issues/1)
 
-### LatencyFleX issues
+### Game crashes on launch with Cause: Access Violation Data Execution Prevention (DEP) at: 0x00000000
 
-Some users have reported issues with enabling LatencyFleX. If you see `"Unable to load LatencyFleX library, LatencyFleX disabled."` in your logs, try adding `latencyflex_wine.dll` to your `bin/x64_retail` folder.
+**Steam/Steam Deck:** Try running Northstar with the latest release of [NorthstarProton](https://github.com/cyrv6737/NorthstarProton/releases/latest).
 
-### Game crashes on launch with Cause: Access Violation Data Execution Prevention (DEP) at: 0x00000000
+**Lutris**: Try running with [ProtonGE](https://github.com/GloriousEggroll/proton-ge-custom/). In particular, `Proton-7.3-GE-1` (not `GE-Proton7-3`) has so far been found to be reliable in this case. Check this [Github issue comment for a guide](https://github.com/R2Northstar/Northstar/issues/1#issuecomment-1062483190).
```

Comment:
Wait, I'm confused. I thought in discussion in `#linux` on the Northstar Discord the consensus between you and @cyrv6737 was to not recommend Proton outside of Steam? And if we do, what stops us from going straight to NorthstarProton? ^^

Reply:
Ok, I missed a few messages, NorthstarProton specifically is not recommended outside of Steam. Not sure what the stance on Proton in general is though ^^

Reply:
Proton in general is also not quite recommended with Lutris, but the previous version of this guide made no distinction. How about I change this to link back to the install section saying it's been resolved recently?

Another option as I mention wine-tkg as a backup for Lutris here instead.

Reply:
@GeckoEidechse @KyleGospo 

There should be no recommendations at all to use any build of Proton outside of Steam. If my words are not enough please see these two posts made by GloriousEggroll.

https://www.reddit.com/r/linux_gaming/comments/uzrz2k/a_thread_about_using_protonge_and_winege_builds/

https://www.reddit.com/r/linux_gaming/comments/w0vfn1/another_protonge_vs_winege_thread/

Lutris Wine should be enough to run Northstar, however if we need to recommend another option incase of the Access Violation DEP error, wine-tkg is a fine recommendation as long as it is *just* wine-tkg and not proton-tkg.